x-pack/plugins/reporting/server/lib/event_logger/types.ts

```diff
@@ -5,31 +5,23 @@
  * 2.0.
  */
 
+import { LogMeta } from 'src/core/server';
 import { ActionType } from './';
 
-type ActionKind = 'event' | 'error' | 'metrics';
-type ActionOutcome = 'success' | 'failure';
-
-interface ActionBase<
-  A extends ActionType,
-  K extends ActionKind,
-  O extends ActionOutcome,
-  EventProvider
-> {
+interface ActionBase<A extends ActionType> {
   event: {
-    action: A;
-    kind: K;
-    outcome?: O;
-    provider: 'reporting';
     timezone: string;
   };
-  kibana: EventProvider & { task?: { id?: string } };
-  user?: { name: string };
-  log: {
-    logger: 'reporting';
-    level: K extends 'error' ? 'error' : 'info';
-  };
   message: string;
+  kibana: {
+    reporting: {
+      actionType?: A;
+      id?: string; // "immediate download" exports have no ID
+      jobType: string;
+      byteSize?: number;
+    };
+    user?: { name: string };
```

Comment:
ECS already [has a user field](https://www.elastic.co/guide/en/ecs/master/ecs-user.html) as [Patrick mentions](https://github.com/elastic/kibana/pull/124762#pullrequestreview-876493044) --  so shouldn't this one be at the top level (not nested under `kibana`), as you've already done in `ReportingEventLogger`?

Reply:
Yep, that's what was intended. :)

3eb1f2e609e